streams/src/main/java/org/apache/kafka/streams/state/internals/AbstractRocksDBSegmentedBytesStore.java

```diff
@@ -185,7 +185,7 @@ public void init(final ProcessorContext context,
         addInvocationRateAndCountToSensor(
             expiredRecordSensor,
             "stream-" + metricScope + "-metrics",
-            metrics.tagMap("task-id", taskName, metricScope + "-id", name()),
+            metrics.storeLevelTagMap(taskName, metricScope, name()),
```

Comment:
Good catch!

Reply:
Also since the "incorrect" metrics name is out in our previous releases, but fixing it we are effectively breaking user's compatibility. I think this okay since we are fixing a bug, but would still worth clarifying it in the upgrade guide docs.

Reply:
Ack